### PR TITLE
run_actions_on_self_hosted

### PR DESCRIPTION
**Pull request by nachog00**
For the self-hosted runner to pick up on actions jobs, they must have the property:

runs-on: self-hosted

I changed only the jobs on CI.yaml for now.

### CODE REVIEW

**Review by zancas — DISMISSED** [review]

**Comment by zancas** [comment]
Hi @nachog00 is this PR still active?  Or has it been superceded?

**Comment by nachog00** [comment]
> Hi @nachog00 is this PR still active? Or has it been superceded?

no. When self hosted runners handle jobs correctly, merging this will make all jobs be picked up by them.

File: .github/workflows/test.yaml [hunk 1]
```diff
@@ -30,7 +32,10 @@ jobs:
           version: 0.9.78
 
       - name: Install protoc
-        run: sudo apt-get install protobuf-compiler
+        run: |
+          if ! command -v protoc; then
+             sudo apt-get install -y protobuf-compiler
+          fi
```
Comment (fluidvanadium) [inline .github/workflows/test.yaml]: since sudo doesnt work in a container, this line needs to go

**Review by fluidvanadium — CHANGES_REQUESTED** [review]

**Review by fluidvanadium — APPROVED** [review]
buenok

**Review by zancas — APPROVED** [review]

**Comment by zancas** [comment]
Since this is blocked on CI... and since it should speed up CI, I am going to merge it.

**Pull request merged**